app/controllers/insights_cloud/ui_requests_controller.rb

```diff
@@ -75,6 +75,18 @@ def assign_header(res, cloud_res, header, transform)
       res.headers[new_header] = header_content
     end
 
+    def translate_insights_host
+      Rails.logger.info "translate_insights_host"
+      facet = InsightsFacet.find_by(uuid: params[:uuid])
+      if facet.present?
+        Rails.logger.info "Found InsightsFacet #{params[:uuid]}"
+        redirect_to host_details_page_path(facet&.host_id)
```

Comment:
Nitpick
```suggestion
        redirect_to host_details_page_path(facet.host_id)
```
We know that the facet is always here